src/platform/win32localfilesystem.cpp

```diff
@@ -128,10 +128,10 @@ void Win32LocalFileSystem::Get_File_List_From_Dir(Utf8String const &subdir,
                     if (name != "." && name != "..") {
                         Utf8String filepath;
                         filepath += subdir;
-                        filepath += name;
-                        filepath += '\\';
+                        filepath.Concat(name);
```

Comment:
Looks like a step back. Why would we want one line += and the other Concat? Wouldn't consistency be better here?